deploy/examples/filesystem-ec.yaml

```
@@ -1,3 +1,15 @@
+# create default csi subvolume group
+apiVersion: ceph.rook.io/v1
+kind: CephFilesystemSubVolumeGroup
+metadata:
+  name: myfs # lets keep the svg crd name same as filesystem name for the default csi svg
```

Using `myfs-csi` might be better because it IDs the fs name as well as the svg name.

```suggestion
  name: myfs-csi # lets keep the svg crd name same as filesystem name for the default csi svg
```